src/components/AppLayout/Header/index.tsx

```diff
@@ -21,6 +21,7 @@ import { getNetworkId } from 'src/config'
 
 const HeaderComponent = (): React.ReactElement => {
   const provider = useSelector(providerNameSelector)
+  const networkId = getNetworkId()
```

Comment:
Same result in the end but we should probably use `useSelector(currentChainId)` here for uniformity.